app/models/volunteer.rb

```diff
@@ -13,4 +13,6 @@ class Volunteer < ApplicationRecord
   validates :ok_to_mail, inclusion: { in: [true, false] }
   validates :underage, inclusion: { in: [true, false] }
   validates :user_id, presence: true
+
+  belongs_to :user
```

Comment:
should this be has_one :user? 
user belongs_to :volunteer, right?

Reply:
@maebeale Good question. I just checked the schema and the foreign key is [in the `volunteers` table](https://github.com/rubyforgood/voices-of-consent/blob/d530386cf8a57ebfc50d60485d9a15d54420efd7/db/schema.rb#L275), so the `belongs_to` has to be on the `Volunteer` side of the relationship.

Reply:
@jgaskins, thanks! per the ERD, volunteer_id should've been placed on user, but this can work too. my sense was that there will be volunteers who never log in, so the primary record shouldn't be user (that's why we had email on Volunteer originally).

to fully implement User has_one :volunteer, my vote would be to display user email on the Volunteer form. and, yeah, we can explain to the stakeholder that everyone in the system has to have a user account first for them to be able to store info about the person. it'll work.